ansible/group_vars/eos/eos.yml

```diff
@@ -1,3 +1,4 @@
 # snmp variables
-snmp_rocommunity: public
+snmp_rocommunity: strcommunity
```

Comment:
This change was done by automatic syncing process. So I didn't make this change, but it was synced from our private repo. I think we need to make the automatic sync our private repo for the test role too.